modules/openy_traction_rec_import/src/Importer.php

```diff
@@ -143,6 +141,9 @@ public function __construct(
     $this->isEnabled = (bool) $settings->get('enabled');
     $this->isBackupEnabled = (bool) $settings->get('backup_json');
     $this->backupLimit = (int) $settings->get('backup_limit');
+
+    $this->siteAliasManager = Drush::service('site.alias.manager');
```

Comment:
this is a constructor, if service is not available - all site will become down after cache clear and Drupal container rebuild. Let's check if service available before obtaining it from container

Reply:
Removed.